app/services/login/after_login_actions.rb

```diff
@@ -21,6 +21,10 @@ def perform
       update_account_login_stats(login_type)
       id_mismatch_validations
 
+      if Flipper.enabled?(:reactivation_experiment_initial_gate, current_user)
+        AcceptableVerifiedCredentialAdoptionService.new(@current_user).perform
```

Comment:
Also, I don't think we need the `@` on `current_user` here.

Reply:
Changed!